content/de/pressreview/2017-03-14_computerworld.md

```diff
@@ -0,0 +1,6 @@
+Title: Internetzensur an der Universität Fribourg
+Slug: 2017-03-14_watson
```

Comment:
Yes, jetzt ist gut: wäre es noch möglich, den Herausgeber (=die Zeitung) auf der Übersichtsseite anzuzeigen, z. B. in Klammern, nach dem Titel?